cypress/support/fragments/data-export/dataExportResults.js

```diff
@@ -13,5 +13,24 @@ export default {
       }),
       MultiColumnListCell({ 'row': 0, 'column': defaultJobProfile }).exists(),
     ]);
-  }
+  },
+
+  verifySuccessExportResultCells() {
+    const dateString = /\d{1,2}\/\d{1,2}\/\d{4},\s\d{2}:\d{2}\s\w{2}/gm;
+    return cy.do([
+      MultiColumnListCell({ 'row': 0, columnIndex: 1 }).is({ content: 'Completed' }),
+      MultiColumnListCell({ 'row': 0, columnIndex: 2 }).perform(element => {
+        expect(element.innerText).to.match(/\d{1,4}/gm);
+      }),
+      MultiColumnListCell({ 'row': 0, columnIndex: 3 }).is({ content: '' }),
+      MultiColumnListCell({ 'row': 0, columnIndex: 4 }).is({ content: defaultJobProfile }),
+      MultiColumnListCell({ 'row': 0, columnIndex: 5 }).perform(element => {
+        expect(element.innerText).to.match(dateString);
+      }),
+      MultiColumnListCell({ 'row': 0, columnIndex: 6 }).is({ content: 'DIKU ADMINISTRATOR' }),
+      MultiColumnListCell({ 'row': 0, columnIndex: 7 }).perform(element => {
```

Comment:
I hop you can catch job id to finish this step fully. "ID of the export job". And what is the meaning of "The records are correctly formatted"? is it related with format of jobId or all row?

Reply:
>  And what is the meaning of "The records are correctly formatted"?

This about whole table formating. And in this tests definetly we don't want to check whole table records